common/table.js

```diff
@@ -239,7 +239,34 @@
                     return vm.getDisabledTuples ? vm.getDisabledTuples(page, vm.pageLimit) : '';
                 }).then(function (rows) {
                     if (rows) vm.disabledRows = rows;
-                    vm.rowValues = DataUtils.getRowValuesFromPage(vm.page);
+                    var rowValues = DataUtils.getRowValuesFromPage(vm.page);
+                    var cellLimit = 500;
```

Comment:
This is not really important, but can you please define this in the `tableConstants` that is at the top of this file?

Reply:
Oh right, I meant to move this to the constants object, especially because it shouldn't ever change.